deps/rabbit/src/rabbit_amqqueue.erl

```diff
@@ -119,21 +118,6 @@
          active, activity_status, arguments]).
 -define(KILL_QUEUE_DELAY_INTERVAL, 100).
 
-warn_file_limit() ->
-    DurableQueues = find_recoverable_queues(),
-    L = length(DurableQueues),
-
-    %% if there are not enough file handles, the server might hang
-    %% when trying to recover queues, warn the user:
-    case file_handle_cache:get_limit() < L of
-        true ->
-            rabbit_log:warning(
-              "Recovering ~tp queues, available file handles: ~tp. Please increase max open file handles limit to at least ~tp!",
```

Comment:
Maybe there can still be a warning at boot-up time that the FD limit looks low. If the limit is =< 1024 for example it's likely that the limit needs to be increased.